### DISCUSSION TITLE
macos编译报错

### DISCUSSION THREAD

**Discussion by Cabriter** [ejoy/ant#34] [Q&A]
更新完submodule,编译luamake后在ant目录下执行luamake报错.
macos: 14.1.2
lua: 5.4.6
![image](https://github.com/ejoy/ant/assets/12536807/73e8bab2-03f1-447e-bfa7-05183b51d4cd)

**Comment by actboy168** [comment]
需要更新luamake

**Reply by Cabriter** [reply]
目前编译的是最新版本的luamake
![image](https://github.com/ejoy/ant/assets/12536807/dc05b6af-7f9c-4629-8319-7e173203783b)

**Reply by Cabriter** [reply]
刚执行错是build了,应该是install,不过也是会是同样的结果.
![image](https://github.com/ejoy/ant/assets/12536807/d297d73c-c0b4-46af-a50c-7fb0e55bef2a)

**Reply by actboy168** [reply]
没有拉子模块

**Reply by Cabriter** [reply]
在build之前都是拉取过的
![image](https://github.com/ejoy/ant/assets/12536807/edbf270f-6ed5-4917-a00c-d58829ec9026)

**Reply by actboy168** [reply]
错误信息就是说`3rd/bgfx.luamake/use.lua`这个文件不存在。你可以自己检查下，它为什么不存在。

**Reply by Cabriter** [reply]
ok,确认了,是git submodule的问题,submodule update没有拉去数据,但是数据不完整,把3rd删掉重新拉了以后就好了. : )
